spring-cloud-deployer-local/src/test/java/org/springframework/cloud/deployer/spi/local/LocalDeployerPropertiesTests.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright 2019 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.cloud.deployer.spi.local;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.boot.context.properties.EnableConfigurationProperties;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.junit4.SpringRunner;
+
+/**
+ * @author David Turanski
+ **/
+@SpringBootTest(classes = LocalDeployerPropertiesTests.TestConfig.class,
+properties = {"spring.cloud.deployer.local.maximumConcurrentTasks=2",
+	"spring.cloud.deployer.local.platformMaximumConcurrentTasks=1"})
+
+@RunWith(SpringRunner.class)
+public class LocalDeployerPropertiesTests {
+	@Autowired LocalDeployerProperties deployerProperties;
+	@Test
+	public void testMaxConcurrentPropeties() {
+
+		//deployerProperties.setShutdownTimeout(-999);
```

Comment:
Did you forget to actually implement boot binding tests?

Reply:
Check i.e. `RandomPortRangeContextTests` how boot makes it easy to test these bindings.

Reply:
Whoops. This test was left over from a previous incarnation. It's no longer needed